geode-docs/getting_started/intro_to_clients.html.md.erb

```diff
@@ -0,0 +1,357 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<% set_title("Introduction to", product_name_long, "Clients") %>
+
+This section provides basic starting points for a variety of <%=vars.product_name_long %> clients, along
+with very rudimentary connect, put, get operations, and then a reference to more in-depth docs and examples on
+how to use the client.
+
+For an in-depth look at how to use <%=vars.product_name_long%> clients, see [Client/Server Configuration](../topologies_and_comm/cs_configuration/chapter_overview.html).
+
+## <a id="client-intro-installing"></a>Installing <%=vars.product_name_long%>
+
+You can download <%=vars.product_name_long%> from the website, run a Docker image, or install with homebrew on OSX.
+See [How to Install Apache Geode](installation/install_standalone.html) for details.
+
+## <a id="client-intro-starting"></a>Starting an <%=vars.product_name_long%> Cluster
+
+For client tests and examples, start a simple cluster and create an example region.
+
+Start an <%=vars.product_name_long%> cluster with one locator and one server.
+
+```
+$ gfsh 
+gfsh> start locator
+gfsh> start server
+```
+
+Create a region called "helloWorld".
+
+```
+gfsh> create region --name=helloWorld --type=PARTITION 
+```
+
+When you are through running client tests and examples, shut down the <%=vars.product_name_long%> cluster:
+
+```
+gfsh> shutdown --include-locators=true
+```
+
+## <a id="client-intro-java"></a><%=vars.product_name_long%> Java Client
+
+**Maven Dependencies**
+
+Declare the following dependency and replace $VERSION, with the version of <%=vars.product_name_long%> that you have installed.
+
+```
+<dependencies>
+     <dependency>
+        <groupId>org.apache.geode</groupId>
+        <artifactId>geode-core</artifactId>
+        <version>$VERSION</version>
+    </dependency>
+</dependencies>
+```
+
+**Gradle Dependencies**
+
+```
+dependencies {
+  compile "org.apache.geode:geode-core:$VERSION"
```

Comment:
`compile` is deprecated and will be removed in the next version of gradle.  `implementation` should be used

Reply:
corrected.